tests/test_visitors/test_ast/test_complexity/test_counts/test_context_manager_targets.py

```diff
@@ -0,0 +1,57 @@
+# -*- coding: utf-8 -*-
+
+import pytest
+
+from wemake_python_styleguide.violations.complexity import (
+    TooManyContextManagerAssignmentsViolation,
+)
+from wemake_python_styleguide.visitors.ast.complexity.counts import (
+    ContextManagerVisitor,
+)
+
+incorrect_count = """
+with open('') as first, second:
+    ...
+"""
```

Comment:
We also need to test this case:

```python
with open('first') as first, open('second') as second:
    ...
```

Reply:
It should be allowed.

Reply:
@geoc0ld on the other hand, no. This should not be allowed. This is very hard to read.
Please, write a test that this sample raises a violation.

Reply:
@geoc0ld on the other hand, no. This should not be allowed. This is very hard to read.
Please, write a test that this sample raises a violation.